src/components/Pipelines/PipelinesList.vue

```diff
@@ -86,8 +86,7 @@ export default {
   },
   methods: {
     ...mapMutations('pipelines', ['addIdPipeline', 'removebyIdPipelines', 'updateSelectedPipelines']),
-    ...mapMutations('target', ['removeTargetEntityToDelete']),
-    ...mapMutations('general', ['addEntityToDelete']),
+    ...mapMutations('general', ['addEntityToDelete', 'removeTargetEntityToDelete']),

```

Comment:
why do we have on Agent component to use this target method removeTargetEntityToDelete ??